explorer/src/entity/JobRun.ts

```diff
@@ -115,7 +118,25 @@ export const search = async (
     query = query.offset(offset)
   }
 
-  return query.orderBy('job_run.createdAt', 'DESC').getMany()
+  return query
+    .leftJoinAndSelect('job_run.chainlinkNode', 'chainlink_node')
+    .orderBy('job_run.createdAt', 'DESC')
+    .getMany()
+}
+
+type Modify<T, R> = Pick<T, Exclude<keyof T, keyof R>> & R
+
+type JobRunPresenter = Modify<
+  JobRun,
+  {
```

Comment:
Having a type literal here makes it a bit harder to read, maybe lift this out? e.g:

```
type Presenter ...
type JobRunPresenter = Modify<JobRun, Presenter>...
```